budget_manager/views.py

```diff
@@ -1,3 +1,61 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.views import generic, View
+from django.urls import reverse_lazy
+
+from budget_manager.forms import IncomeForm
+from budget_manager.models import Income
 
 # Create your views here.
+
+def income_added(request):
+    return render(request, 'income_added.html', context={})
+
+def income_update(request):
+    return render(request, 'income_update.html', context={})
+
+def income_delete(request):
+    return render(request, 'income_delete_2.html', context={})
+
+
+class IncomeCreateView(generic.FormView):
```

Comment:
Ciekawe, że to już drugi taki przypadek, skoro robisz `CreateView` i jest coś takiego jak `CreateView` w widokach generycznych to użyjmy tego

https://docs.djangoproject.com/en/4.2/ref/class-based-views/generic-editing/#django.views.generic.edit.CreateView